templates/ribbon.html

```diff
@@ -0,0 +1,10 @@
+<div class="ribbon">
```

Comment:
It's a HTML file, so I guess more appropriate would be to put some HTML tags first and maybe put this class into a `body`?

Reply:
Why?

The HTML files would include this piece of code. Adding another <html> would break the inheritor file.